phpunit.xml.dist

```diff
@@ -8,7 +8,6 @@
          convertWarningsToExceptions="true"
          processIsolation="false"
          stopOnFailure="false"
-         syntaxCheck="false"
```

Comment:
I'd also update all these switches and add xsi schema, you can copy those from DBAL: https://github.com/doctrine/dbal/blob/master/phpunit.xml.dist#L14